module/reports/helper.js

```diff
@@ -101,63 +102,7 @@ module.exports = class ReportsHelper {
 				projectReport[CONSTANTS.common.SUBMITTED_STATUS] = 0
 				projectReport[CONSTANTS.common.INPROGRESS_STATUS] = 0
 				projectReport[CONSTANTS.common.STARTED] = 0
-
-				let types = await this.types()
-				let returnTypeInfo = types.data.filter((type) => {
-					if (type.value == reportType) {
-						return type.label
-					}
-				})
-				// if (!projectDetails.length > 0) {
-				//Initially not supporting pdf report, hence resolving response here. Remove resolve if pdf capability needs to be added
-				// return resolve({
-				// 	message: CONSTANTS.apiResponses.REPORTS_DATA_NOT_FOUND,
-				// 	data: {
-				// 		dataAvailable: false,
-				// 		data: {
-				// 			categories: categories,
-				// 			tasks: tasksReport,
-				// 			projects: projectReport,
-				// 		},
-				// 	},
-				// })
-
-				if (getPdf == true) {
```

Comment:
@MallanagoudaB what happen if we need pdf?

Reply:
Its generating below line no 195 the logic is same

Reply:
@MallanagoudaB I think here we had two type reports entity type reports and project report. In this PR https://github.com/ELEVATE-Project/project-service/pull/23/files#diff-13ab73c9e43ba8f09171958cd34a636841985118315ae882a151e900cd17a009

You completely ignored the entity report . Hence the entire logic was removed I was not able catch it during the review. Is there any particular reason why we removed that?